lib/widgets/shopping_list_tile.dart

```diff
@@ -1,25 +1,22 @@
 import 'package:flutter/material.dart';
-import 'package:to_do_flutter/shoppingList.dart';
+import 'package:to_do_flutter/screens/product_list.dart';
 
-class MyBox extends StatefulWidget {
-  final List<String> myShoppingLists;
+class MyTile extends StatefulWidget {
+  final String title;
   final int myIndex;
   final Function(int) deleteFunction;
```

Comment:
Przy podawaniu parametru wiemy, że to funkcja, więc ta nazwa chyba będzie trochę lepsza
```suggestion
  final Function(int) onDelete;
```